src/components/profile/rockets/MyProfile.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import '../MyProfile.scss';
+import RocketProfile from './RocketProfile';
+
+const MyProfile = () => {
+  const { rockets } = useSelector((store) => store.rockets);
+  // console.log(rockets)
+  const rocketsStatus = rockets.filter((rocket) => rocket.reserved);
+
+  return (
+    <div className="profile-container">
+      {/* show missions on profile page */}
```

Comment:
- [ ] **`[REQUIRED]`** Same here, the unnecessary portions of code commented in your codebase. please consider.